test/modules/TestResultCoordinator/Services/TestResultReportingService.cs

```diff
@@ -103,7 +103,10 @@ async void DoWorkAsync(object state)
                 {
                     Uri blobContainerWriteUriForLog = await TestReportUtil.GetOrCreateBlobContainerSasUriForLogAsync(this.serviceSpecificSettings.StorageAccountConnectionString);
                     blobContainerUri = $"{blobContainerWriteUriForLog.Scheme}{Uri.SchemeDelimiter}{blobContainerWriteUriForLog.Authority}{blobContainerWriteUriForLog.AbsolutePath}";
-                    await TestReportUtil.UploadLogsAsync(Settings.Current.IoTHubConnectionString, blobContainerWriteUriForLog, this.logger);
+
+                    // Add buffer to duration to ensure we capture all logs
+                    TimeSpan uploadLogDuration = this.sendReportFrequency + TimeSpan.FromMinutes(10);
```

Comment:
It looks like `this.sendReportFrequency` is set to -1ms in the connectivity case. Will this math (-1ms + 10min) cause problems?

Reply:
Ah yes this will cause issues. Thinking about the best fix now.